src/AdminTab/AdminTab.tsx

```diff
@@ -42,7 +43,13 @@ export const AdminTab: FunctionComponent<AdminTabProps> = ({
         <div className="AdminTab-tables">
           <div className="AdminTab-tables-header">
             <Title level={3}>{t('Imported Tables')}</Title>
-            <Button onClick={onOpenImportDataTab} type="primary" size="large" icon={<PlusOutlined />}>
+            <Button
+              onClick={onOpenImportDataTab}
+              type="primary"
+              size="large"
+              disabled={postgresqlStatus !== ServiceStatus.Running}
```

Comment:
1. should we add a tooltip to explain why it's disabled?
2. should we add the same functionality to the final "import" button in the `ImportStep`?

Reply:
> should we add a tooltip to explain why it's disabled?

Postgres status will have a red cross which should be obvious enough I think.

> should we add the same functionality to the final "import" button in the ImportStep?

We could, but I think 95% of the time when postgres is down it will be because it failed to boot up. In that case the main page will already prevent us from going to the import flow.

I don't see how it can go from a running service to a broken service without the user killing the process manually by pid or task manager.

Reply:
> > should we add a tooltip to explain why it's disabled?
> 
> Postgres status will have a red cross which should be obvious enough I think.

Obvious to us, but we should also speak to users not interested in what PostgreSQL is responsible for under the hood.

> I don't see how it can go from a running service to a broken service without the user killing the process manually by pid or task manager.

Makes sense. The user would be confronted with the error message after hitting the button, and likely we have a bigger problem on our hands if that happens,. Not a lot of gain in preventing that confrontation.

Reply:
> Obvious to us, but we should also speak to users not interested in what PostgreSQL is responsible for under the hood.

You're right. How about we keep the button available but show a message saying postgres is not ready when clicked? I read somewhere that it's better design to allow interaction but give feedback on why you can't do something.

Reply:
> better design to allow interaction but give feedback on why you can't do something

Oh, I didn't realize that. I need to pay attention to find out what's my preference, actually...  the UI underneath my fingers now (github) prefers to disable & explain (e.g. merging branch with conflict).

Anyway, it sounds good too!